.github/workflows/node-ci-main.yml

```diff
@@ -33,25 +33,68 @@ jobs:
 
   coverage:
     needs: [prime_cache_primary]
-    name: Gather coverage
+    name: Jest Coverage
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # Use a matrix as it means we get the version info in the job name
-        # which is very helpful.
         os: [ubuntu-latest]
         node-version: [16.x]
     steps:
     - uses: actions/checkout@v2
+
     - name: Use Node.js ${{ matrix.node-version }} & Install & cache node_modules
       uses: Khan/actions@shared-node-cache-v0
       with:
         node-version: ${{ matrix.node-version }}
-    # Collect and upload coverage
-    - name: Run Jest with coverage
+
+    - name: Jest with coverage
       run: yarn coverage
+
     - name: Upload Coverage
-      uses: codecov/codecov-action@v3
+      # We don't bother collecting a record of coverage for dependabot changes
+      if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
I don't think we do dependabot on this repo yet, but I'm fine leaving this here for if/when we do start using it.